source/SharedCrtResourceManager.h

```diff
@@ -42,7 +43,8 @@ namespace Aws
                 std::shared_ptr<Crt::Mqtt::MqttConnection> connection;
                 aws_allocator *allocator{nullptr};
                 aws_mem_trace_level memTraceLevel{AWS_MEMTRACE_NONE};
-                std::vector<Feature *> *features;
+                std::shared_ptr<Util::FeatureRegistry> features;
+                // std::vector<Feature *> *features;
```

Comment:
Remove commented code, we have version control. I see this in other places in this review, but won't repeat comment.